index.html

```diff
@@ -176,10 +311,55 @@ <h2 class="work-title">Multi-Post Stories</h2>
           <li>Javascript</li>
         </ul>
         <div>
-          <button type="button" class="btn">See Project</button>
+          <button data-modal-target="#modal" class="btn">See Project</button>
+
+          <div class="modal" id="modal">
+
+            <div class="modal-description">
+              <h2 class="work-title">tonic</h2>
+              <button data-btn-close class="btn-close">&times;</button>
+            </div>
+
+            <div class="modal-body">
+              <ul class="canopy">
+                <li>canopy</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+              <div class="work-img">
+                <img src="images/4.svg" alt="tonic image" />
+              </div>
+              <p class="work-details">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required.
+              </p>
+              <ul class="work-tools">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+              </ul>
+              <div>
+                <button type="button" class="btn">
+                  <span class="btn-text">See live</span>
+                  <span class="btn-icon">
+                    <img src="images/live-icon.svg" alt="Icon">
+                  </span>
+                </button>
+                <button type="button" class="btn">
+                  <span class="btn-text">See source</span>
+                  <span class="btn-icon">
+                    <img src="images/github-icon.svg" alt="github-Icon">
+                  </span>
+                </button>
+              </div>
+            </div>
+
+
+          </div>
         </div>
       </div>
     </div>
+    <div id="overlay"></div>
   </section>
```

Comment:
- [ ] As suggested by the previous code reviewer, I kindly suggest that already created HTML code for the projects list should be removed, and instead it should be added via JavaScript.